scripts/spack/configs/blueos_3_ppc64le_ib_p9/compilers.yaml

```diff
@@ -45,11 +45,11 @@ compilers:
     modules: []
     operating_system: rhel7
     paths:
-      cc:  /usr/tce/packages/xl/xl-2019.08.20/bin/xlc
-      cxx: /usr/tce/packages/xl/xl-2019.08.20/bin/xlC
-      f77: /usr/tce/packages/xl/xl-2019.08.20/bin/xlf2003
-      fc:  /usr/tce/packages/xl/xl-2019.08.20/bin/xlf2003
-    spec: xl@16.1.1.1
+      cc: /usr/tce/packages/xl/xl-2020.08.24-cuda-11.0.2/bin/xlc_r
```

Comment:
What does the `_r` mean?

Reply:
> What does the `_r` mean?

Looking at the file, `xlc_r` is a wrapper script for `xlc`.
Wrapper may not be necessary, I 'll give that a try.
I took these compiler specs from [radiuss](https://github.com/LLNL/radiuss-spack-configs/blob/e58c9fbfc68d41c997fd33b8d9ec7058335ea4a6/blueos_3_ppc64le_ib/compilers.yaml)

Reply:
I believe `_r` means 'reentrant', which causes the compiler to use thread safe versions of functions in the standard library.